setup.py

```diff
@@ -2,16 +2,16 @@
 
 setup(
     name='ConferenceScheduler',
-    version='',
+    version='1.0.0',
     packages=[
         'conference_scheduler',
     ],
-    url='',
+    url='https://github.com/PyconUK/ConferenceScheduler',
     license='MIT',
     author='Owen Campbell, Vince Knight',
     author_email='Owen Campbell <owen.campbell@tanti.org.uk>',
-    description='',
-    install_requires=['pulp'],
+    description='A Python tool to assist the task of scheduling a conference',
+    install_requires=['pulp', 'numpy'],
     setup_requires=['pytest-runner'],
     tests_require=['pytest', 'pytest-pep8']
```

Comment:
I think we could add something like:

```
 23     },                                                                                                                                            
 24     classifiers=[                                                                                                                                 
 26         'Programming Language :: Python :: 3.6',                                                                                                  
 28         ],                                                                                                                                        
 29     python_requires='>=3.6',                                                                                                                      
 30 )     
```
This would mean that if someone tries to pip install it with 3.5 they get a meaningful error message.

(This is what we have in Axelrod: https://github.com/Axelrod-Python/Axelrod/blob/master/setup.py)